cps/templates/index.xml

```diff
@@ -15,101 +15,101 @@
   </author>
   <entry>
     <title>{{_('Hot Books')}}</title>
-    <link href="{{url_for('opds.feed_hot')}}" type="application/atom+xml;profile=opds-catalog"/>
+    <link href="{{url_for('opds.feed_hot')}}" type="application/atom+xml;profile=opds-catalog;kind=acquisition"/>
     <id>{{url_for('opds.feed_hot')}}</id>
     <updated>{{ current_time }}</updated>
     <content type="text">{{_('Popular publications from this catalog based on Downloads.')}}</content>
   </entry>
   <entry>
     <title>{{_('Top Rated Books')}}</title>
-    <link href="{{url_for('opds.feed_best_rated')}}" type="application/atom+xml;profile=opds-catalog"/>
+    <link href="{{url_for('opds.feed_best_rated')}}" type="application/atom+xml;profile=opds-catalog;kind=acquisition"/>
     <id>{{url_for('opds.feed_best_rated')}}</id>
     <updated>{{ current_time }}</updated>
     <content type="text">{{_('Popular publications from this catalog based on Rating.')}}</content>
   </entry>
   <entry>
     <title>{{_('Recently added Books')}}</title>
-    <link href="{{url_for('opds.feed_new')}}" type="application/atom+xml;profile=opds-catalog"/>
+    <link href="{{url_for('opds.feed_new')}}" type="application/atom+xml;profile=opds-catalog;kind=acquisition"/>
     <id>{{url_for('opds.feed_new')}}</id>
     <updated>{{ current_time }}</updated>
     <content type="text">{{_('The latest Books')}}</content>
   </entry>
   <entry>
     <title>{{_('Random Books')}}</title>
-    <link href="{{url_for('opds.feed_discover')}}" type="application/atom+xml;profile=opds-catalog"/>
+    <link href="{{url_for('opds.feed_discover')}}" type="application/atom+xml;profile=opds-catalog;kind=acquisition"/>
     <id>{{url_for('opds.feed_discover')}}</id>
     <updated>{{ current_time }}</updated>
     <content type="text">{{_('Show Random Books')}}</content>
   </entry>
   {% if not current_user.is_anonymous %}
   <entry>
     <title>{{_('Read Books')}}</title>
-    <link href="{{url_for('opds.feed_read_books')}}" type="application/atom+xml;profile=opds-catalog"/>
+    <link href="{{url_for('opds.feed_read_books')}}" type="application/atom+xml;profile=opds-catalog;kind=acquisition"/>
     <id>{{url_for('opds.feed_read_books')}}</id>
     <updated>{{ current_time }}</updated>
-    <content type="text">{{_('Read Books')}}</content>
+    <content type="text">{{_('Show Read Books')}}</content>
   </entry>
```

Comment:
The text is identical to the text in the "normal" user interface. There was recently a issue about different text between UI and opds feed. Furthermore all other text are also not saying showing like Publisher, Categories and so on -> no change